### ISSUE TITLE
Complete cImport symbols 

### ISSUE THREAD

**Issue by rcorre** [zigtools/zls#313]
> Notable language features that are not currently implemented include @cImport

(from the README).

Is this a feature `zls` eventually wants to have? If so, what would it take? Is it blocked on https://github.com/ziglang/zig/issues/4045?

**Comment by alexnask** [comment]
Hi.  
To implement this feature we would need something like a compiler option that prints the location of each cImport and the corresponding generated zig file in the cache.  
I will get around to adding this to the compiler at some point

**Comment by track3r** [comment]
Very nice to have for game development.

**Comment by InKryption** [comment]
@alexnask Would `zig build [...] --verbose-cimport` meet the requirements for this? Or `std.build.Builder`'s equivalent `verbose_cimport` field.

**Comment by InKryption** [comment]
Don't mean to nag or to annoy, but I really would like to know: is there a path forward for this using `--verbose-cimport`/`Builder.verbose_cimport`? If there is, I would be happy to try my hand at making an attempt, if anyone could suggest where to start. Is there a stub waiting to have this implemented or anything?
